drivers/mcp2515/include/mcp2515_params.h

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright (C) 2016 OTA keys S.A.
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     drivers_mcp2515
+ * @{
+ *
+ * @file
+ * @brief       Parameters for the CAN driver implementation
+ *
+ * @author      Vincent Dupont <vincent@otakeys.com>
+ * @}
+ */
+
+#ifndef MCP2515_PARAMS_H
+#define MCP2515_PARAMS_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include "can/device.h"
+#include "candev_mcp2515.h"
+
+#include "board.h"
+
+/**
+ * @name    Set default configuration parameters for the MCP2515
+ * @{
+ */
+#ifndef MCP2515_PARAM_SPI
+#define MCP2515_PARAM_SPI SPI_DEV(0)
+#endif
+
+#ifndef MCP2515_PARAM_SPI_MODE
+#define MCP2515_PARAM_SPI_MODE SPI_MODE_0
+#endif
+
+#ifndef MCP2515_PARAM_SPI_CLK
+#define MCP2515_PARAM_SPI_CLK SPI_CLK_10MHZ
+#endif
+
+#ifndef MCP2515_PARAM_CS
+#define MCP2515_PARAM_CS SPI_HWCS(0)
+#endif
+
+#ifndef MCP2515_PARAM_RST
+#define MCP2515_PARAM_RST GPIO_PIN(0, 0)
+#endif
+
+#ifndef MCP2515_PARAM_INT
+#define MCP2515_PARAM_INT GPIO_PIN(0, 1)
+#endif
+
+#ifndef MCP2515_PARAM_CLK
+#define MCP2515_PARAM_CLK (8000000ul)
```

Comment:
This is the external clock frequency (the frequency of the oscillator hooked up to the mcp2515). It is used in the defconfig but can be overwritten in the Makefile (for now, will probably still change because you suggested a more RIOT-like way).

I'll add the same comment as is used in the struct where it is used.